modules/completion/Makefile

```diff
@@ -0,0 +1,10 @@
+BASH_COMPLETION_FLAG ?= ~/.make.bash.completion
+
+completion/bash:
+	$(SELF) -s $(DEFAULT_HELP_TARGET) | cut -d ' ' -f3  | sed 's/\x1b\[[0-9;]*m//g'
+
+## Install completion script for bash
+completion/install/bash:
+	@[ -f $(BASH_COMPLETION_FLAG) ] || echo "complete -W \"\\\`[ -f Makefile ] && make completion/bash \\\`\" make" >> ~/.bashrc
```

Comment:
Does this work in directories that have a `Makefile`, but no `completion/bash` target? I think I ran into this error.

Reply:
(basically, does gracefully disable when not in a `build-harness` compatible make environment)